ddtrace/appsec/remoteconfiguration.py

```diff
@@ -0,0 +1,56 @@
+import os
```

Comment:
This module is in the public API now which means `enable_appsec_rc` and `appsec_rc_reload_features` are public methods. Is this intentional?

Reply:
No it isn't, could i `remoteconfiguration.py` to `remoteconfiguration.py` right?

Reply:
Yeah I think we can make the module internal 👍